src/Calendar/Calendar.js

```diff
@@ -673,19 +696,19 @@ Calendar.displayName = 'Calendar';
 
 Calendar.propTypes = {
     /** Blocks dates that are in between the blocked dates */
-    blockedDates: PropTypes.arrayOf(PropTypes.instanceOf(Date)),
+    blockedDates: PropTypes.arrayOf(PropTypes.instanceOf(moment)),
```

Comment:
The change here, and subsequent areas below, the `Date` was giving me an error in the console while writing tests and was expecting instances of a Moment. This seemed to mitigate the error in the console. Not sure if that will cause a problem 🤷

Reply:
Yeah, fair call. I do remember mixing up `Date` and `moment` props at some point.